src/style/panel/panel.less

```diff
@@ -388,6 +388,18 @@
         .focus-offset;
         .focus-border-radius;
       }
+
+      &:first-child {
+        flex-grow: 1;
+        margin-left: var(--content-title-bar-height);
+        text-align: center;
+      }
+
+      &:last-child {
+        flex-grow: 1;
+        margin-right: var(--content-title-bar-height);
+        text-align: center;
+      }
     }
```

Comment:
this was removed in this PR https://github.com/wireapp/wire-webapp/pull/17240/files#diff-e3ccc1bc86b648d28817b27e6b0a28664ecdafbf0659ef4ce9c38de261870085L391-L402 

It also create other issues, reverting it fixes them